### ISSUE TITLE
Does't work base on cordova Apps

### ISSUE THREAD

**Issue by thejourneyoflife** [angular-translate/angular-translate#876]
This module does not work with cordova apps after deploy the app to a device

**Comment by knalli** [comment]
Please give more details...

**Comment by thejourneyoflife** [comment]
@knalli  When I choose `angular-translate` as my multiple language solution,its works well in the browser,but when i deploy or emulation using xcode emulator,its only display the prop like this 
`app.title` and have not translate

**Comment by knalli** [comment]
So, we cannot investigate without more details. We cannot debug your application to find out what you have configured and what there happens or not happens. 

Perhaps you are using XHR but they failed in your setup.. who knows?

Additional resources: https://github.com/angular-translate/angular-translate/search?q=cordova&type=Issues&utf8=%E2%9C%93

**Comment by RonMen** [comment]
Hi.

Since the other issue was closed I will try here. In iOS running the page on a local server in Safari and also within an iOS app it isn't possible to get translations. So this isn't really a problem of iOS together with Cordova build apps, it seems a problem of in iOS Browser as well.

**Comment by tspaeth** [comment]
When I'm running some of the angular-translate.github.io - demos ("Getting started" - just did a quick test) from within my iOS Browser (Safari) - they work. So this is not true that translate will not work on iOS-devices in general!

**Comment by RonMen** [comment]
OK, think we might go a little deeper into some configs here?

My modules are the following:
'ngRoute','ngAria','ngAnimate','ngMaterial','mobile-angular-ui','pascalprecht.translate',	'ui.bootstrap','leaflet-directive'


My setup for the translations in the config is like this one:
```javascript
...
// german
$translateProvider.translations('de_DE', {
	HEADLINE:  'Eine Überschrift',
	MENU_SETTINGS: 'Einstellungen',
	BTN_BACK: 'Zurück',
	BTN_MENU: 'Menü',
	BTN_CLOSE: 'Schließen',
	BTN_REFRESH: 'Aktualisieren'
});

// english
$translateProvider.translations('en_US', {
	HEADLINE:  'Some Headline',
	MENU_SETTINGS: 'Settings',
	BTN_BACK: 'Back',
	BTN_MENU: 'Menu',
	BTN_CLOSE: 'Close',
	BTN_REFRESH: 'Refresh'
});

$translateProvider.determinePreferredLanguage();
...
```
Within the template I'm using this for example:
```
{{ 'HEADLINE' | translate }}
```
For
This is working great using Chrome browser and also within an Android-App container but it isn't working on other browsers and within an iOS-App container.

I also tried to use something like this definition  shown in the Getting-Started:
```javascript
...
// german
$translateProvider.translations('de', {
...
});

// english
$translateProvider.translations('en', {
...
});

$translateProvider.determinePreferredLanguage();
...
```

Something I'm missing?

Regards

**Comment by tspaeth** [comment]
Did you also try the directive-approach with sth. like <span translate="HEADLINE"></span> ? If not: Please check if that is working.

**Comment by RonMen** [comment]
Hi.

I tried this now and it is also not working in other browsers than Chrome. Something broken because of the modules used?

**Comment by knalli** [comment]
Are you really sure you have not any error in logs?

**Comment by RonMen** [comment]
@knalli no there aren't any errors in the logs. See the attached screenshot.
![client_translation_issue_20150114](https://cloud.githubusercontent.com/assets/2003172/5735337/b4d0da0e-9bc4-11e4-8840-e72152ed094a.jpg)

**Comment by knalli** [comment]
Then you have to debug (i.e. making _Conditional breakpoint actions with console.log_ see in https://developer.chrome.com/devtools/docs/tips-and-tricks) within the translate directive to verify the actual steps.

**Comment by RonMen** [comment]
So. I cleared all Caches and tried again in Chrome, FF on Windows and Safari on MacOSX debugging some breakpoints using german browser interfaces for now:

Chrome only works this way:
```
// german
$translateProvider.translations('de_DE', {
...
});
```

FF after resetting cache works this way:
```
// german
$translateProvider.translations('de', {
...
});
```

Safari on MacOSX after resetting cache works this way:
```
// german
$translateProvider.translations('de_de', {
...
});
```

So if this isn't that strikt it wouldn't be nice since I would need to add a lot translations to support the one language only.

I also tried running it on iOS (simulator only, devices are on the road today) changing the devices language to german. The local of the call is "en_us" (it isn't "en_US"), the navigator.language of the simulator is "en-us" and so it isn't german and only worked this way to get translations, but since device language is german it isn't correct:
```
// english
$translateProvider.translations('en_us', {
...
});
```

For iOS it might be the simulator running into an issue here, I need to try further when the devices are back here. Also to get some Cordova running for test to see if this is also
